guide/daily-spending-wallet/security.md

```diff
@@ -362,6 +371,22 @@ By including UX patterns for hiding information pattern wallets, we give users a
 
 ---
 
+## Preventing theft when offline
+
+Lightning wallets need to be online to monitor their payment channels to prevent their counter parties from attempting to steal their bitcoin. Mobile wallets however are frequently offline. This is usually due to a poor connection or the user not having the wallet open on their device. 
```

Comment:
```suggestion
Lightning wallets need to be online to monitor their payment channels to prevent their counterparties from attempting to steal their bitcoin. Mobile wallets, however, are frequently offline, usually due to a poor connection or the user not having the wallet open on their device.
```

Reply:
[Hemingway](https://hemingwayapp.com) complains about the second, combined sentence being hard to read. It also complains about the first sentence being very hard to read. Wonder if we should do more general revision based on it.

Just a comment, accepting your change anyways.

Reply:
I'm surprised Hemingway would be that opinionated about the changes. "Counterparties" as a single word is just a spelling correction. The commas surrounding "however" are a grammatical necessity.

Perhaps it's upset about replacing "This is" with a comma? I don't have strong feelings about that one. Sometimes too much "this, this, this" can make it difficult to know what "this" refers to, but hopefully it's clear in this context. Maybe Hemingway will be happy if you bring the"this" back?